src/main/java/org/bookstore/dto/UpdateBookDto.java

```diff
@@ -0,0 +1,22 @@
+package org.bookstore.dto;
+
+import jakarta.validation.constraints.Min;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class UpdateBookDto {
```

Comment:
use the same validators

Reply:
I will do it, but I was thinking about the case where it might be possible to send only the necessary fields.